### DISCUSSION TITLE
Using summary_table_fields

### DISCUSSION THREAD

**Discussion by ngms17** [jertel/elastalert2#922] [Q&A] [answered]
Using "aggregation" with "summary_table_fields" to present the results on a table format via email. The table appears unformatted, is there any way to fix this?

Also, is it possible to remove the text "Aggregation resulted in the ... "

![email](https://user-images.githubusercontent.com/19928515/182569472-442ab7a6-d5bf-4bc6-8f45-e0a219b7cead.PNG)

Thank you

**Comment by jertel** [comment, marked as answer]
The formatting problem occurs because your mail reader is using a variable-width font, such as `helvetica`, instead of a fixed-width font, like `courier new` or `system`. Improving this will require someone to submit a PR to this project. The PR could influence the HTML body styling of the email, or generate the table as an image attachment. I suspect most people don't run into this problem because email usage for alerting purposes is losing favor over modern methods, such as chat channels like Slack, which format this type of table into code-blocks that preserves the fixed-width style. Alternatively, if you used an alerting channel that supported markdown, you could then set the `summary_table_table` property to `markdown` instead of the default `ascii`, which would also resolve the problem. 

As far as the "Aggregation resulted in..." header, that is hardcoded and would require a PR to change this as well.

**Reply by jonny-wg2** [reply]
It would be great to have an option to remove or custom set `Aggregation resulted in..."` header
